features/step_definitions/user.rb

```diff
@@ -17,7 +17,7 @@
   visit '/users/sign_in'
   page.fill_in 'Email', with:  @user.email
   page.fill_in 'Password', with:  @user.password
-  page.click_button 'Log in'
+  page.click_button 'Sign in'
 end
 
 Then(/^the user should see an log in success message$/) do
```

Comment:
Agreed, but I didn't write those specs. I assumed there was a reason to write it that way. Or that it was standard.